assets/app/scripts/services/imagestreams.js

```diff
@@ -13,7 +13,7 @@ angular.module("openshiftConsole")
           // Split the name into useful parts, recalculate name to a common form to
           // include self-references within the same IS
           var from = tagsByName[tag.name].spec.from;
-          if (from) {
+          if (from && (from.kind === "ImageStreamImage" || from.kind === "ImageStreamTag")) {
             var splitChar = "";
```

Comment:
Shouldn't need to initialize `splitChar` to a value since it will always be set below.